modules/nf-core/samtools/sormadup/tests/main.nf.test

```diff
@@ -34,6 +34,42 @@ nextflow_process {
         }
     }
 
+    test("sarscov2 - cram") {
+        config "./cram.config"
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                    [id: 'test', single_end: false], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam.bai', checkIfExists: true)
+                ])
+                input[1] = Channel.of([
+                    [ id:'fasta' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                ])
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                {
+                    assert snapshot(
+                        cram(
+                            process.out.cram[0][1],
+                            "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/sarscov2/genome/genome.fasta",
```

Comment:
per comments in Slack. It seems like using `params` should be possible in the `then` block? But seems like maybe it has to do with whether it is defined in the `when`. 

Perhaps:
```
when {
    params {
        testdata_base = params.modules_testdata_base_path
    }
}

...

then{
...
params.testdata_base + "/genomics/sarscov2/genome/genome.fasta"
```

It's a potential workaround, but maybe also we should open and issue in nf-test to see why `params` defined outside `when` are available in some places but not within `then`?

Reply:
Fixed it cfr @maxulysse 's comment on slack